dotcom-rendering/fixtures/generated/fe-articles/Gallery.ts

```diff
@@ -5719,7 +5719,7 @@ export const Gallery: FEArticle = {
 	format: {
 		design: 'GalleryDesign',
 		theme: 'NewsPillar',
-		display: 'ImmersiveDisplay',
+		display: 'StandardDisplay',
 	},
```

Comment:
This is a temporary manual change in the Gallery fixture file.
There's a PR (https://github.com/guardian/content-api-scala-client/pull/441) that updates `content-api-client` so that the Gallery articles are not anymore get an immersive display. After that change is published, and frontend starts using the new version, we should be able to re-generate the fixture for Gallery which gets the retrieves new data from production.